### PR TITLE
example reverted from 64ab3a3

### PR DESCRIPTION
**Pull request by yahu1031**
**- What I did**

Reverted example from 64ab3a3

### CODE REVIEW

**Review by Intiserahmed — APPROVED** [review]
LGTM

File: at_onboarding_flutter/example/lib/main.dart
Comment (thoson-nws) [inline at_onboarding_flutter/example/lib/main.dart:18]: I have a feature which allow switching theme dark/light here.
This PR will remove feature of example.
Could you please check again?

Comment (thoson-nws) [inline at_onboarding_flutter/example/lib/main.dart:18]: I have a feature which allow switching theme dark/light here.
This PR will remove feature of example.
Could you please check again?

Comment (sonle-geekyants) [inline at_onboarding_flutter/example/lib/main.dart]: I have a feature that allows switching theme dark/light here.
This PR will remove the feature of this example.
Could you please check again?

**Comment by sonle-geekyants** [comment]
@yahu1031 could you please check it again?
The example lost the switching theme feature in your PR.

File: at_onboarding_flutter/example/lib/main.dart
Comment (Intiserahmed) [reply at_onboarding_flutter/example/lib/main.dart]: @sonle-geekyants  But if you check your PR of switching theme dark/light it only contain the main.dart file other files are missing that is the reason why we are reverting it.

Comment (sonle-geekyants) [reply at_onboarding_flutter/example/lib/main.dart]: @Intiserahmed It's changed themeMode of MaterialApp and all child/inherited widget will get theme from parent.
I just want user known that OnBoarding also support for dark theme.

Comment (Intiserahmed) [reply at_onboarding_flutter/example/lib/main.dart]: @sonle-geekyants I understand but the Changes you made didn't take in the PR you did. So can you look into your PR.

Comment (yahu1031) [reply at_onboarding_flutter/example/lib/main.dart]: for time being can we merge this or wait?

Comment (Intiserahmed) [reply at_onboarding_flutter/example/lib/main.dart:18]: Merge

Comment (sonle-geekyants) [reply at_onboarding_flutter/example/lib/main.dart]: Can we wait for a review from Sarika? the commit 64ab3a3 belongs to her. It's better if she knows about these changes.
Thank you all!

**Comment by yahu1031** [comment]
@sonle-geekyants Can you please check this now.

**Comment by sonle-geekyants** [comment]
Look good!

**Pull request merged**
